BertecGUI.py

```diff
@@ -9,18 +9,15 @@
 from datetime import datetime
 from BertecMan import Bertec
 from ViconMan import Vicon
-
-from typing import Any, Callable, List, Union
-import csv
-import logging
-from logging.handlers import RotatingFileHandler
+from logger import Logger
```

Comment:
Shouldn't this be `from opensourceleg.utilities.logger import Logger`? Where is the logger.py file?

Reply:
I downloaded the file and put it in the same folder. I couldn't pip install the opensourceleg library, and you also have the logger.py file in your folder with BertecGUI

Reply:
Why couldn't you pip install it? The reason I'm being picky here is its two points of maintenance and we can end up with an "old" verison of logger.py. I think we should not have logger.py at all in this repo and just reference the osl library one.